src/WebEid.AspNetCore.Example/WebEid.AspNetCore.Example.csproj

```diff
@@ -10,6 +10,14 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
```

Comment:
I think you should set x64 target. For best compatibility

Reply:
For the best compatibility it should be AnyCPU while in that case the architecture stays unspecified in IL and it is able to run on any target platform. x64 limits it with 64-bit platforms.

Reply:
We only provide x64 binaries for libdigidocpp.